pkg/path/path.go

```diff
@@ -1,16 +1,12 @@
 package path
 
 const (
-	// ChartsRepositoryRebasePatchesDir is the directory where a synchronize call will generate any conflicts
-	ChartsRepositoryRebasePatchesDir = "rebase/generated-changes"
-	// ChartsRepositoryCurrentBranchDir is a directory that will be used to store your current assets
-	ChartsRepositoryCurrentBranchDir = "original-assets"
-	// ChartsRepositoryUpstreamBranchDir is a directory that will be used to store the latest copy of a branch you want to sync with
-	ChartsRepositoryUpstreamBranchDir = "new-assets"
```

Comment:
No longer required due to the deprecation of sync and simplification of validate.